readme.md

```diff
@@ -44,6 +44,9 @@ Some Linux distributions offer native packages already. These packages are usual
 * Ubuntu PPA: [`master` branch](https://launchpad.net/~openrct2/+archive/ubuntu/master) and [`develop` branch](https://launchpad.net/~openrct2/+archive/ubuntu/nightly) (`develop` branch builds are temporarily on hold due to [missing functionality in bzr](https://bugs.launchpad.net/ubuntu/+source/bzr-git/+bug/1084403))
 * openSUSE OBS: [games/openrct2](https://software.opensuse.org/download.html?project=games&package=openrct2)
 
+Some *BSD operating systems offer native packages. These packages are usually third-party, but we're trying to resolve issues they are facing.
```

Comment:
Asterisks are markup characters in Markdown. You need to put a backslash before it to get a literal asterisk.